playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -30,6 +33,7 @@
     command: oc scale --replicas=0 dc/docker-registry
 
   - name: Create S3 bucket
+    when: aws_create_bucket | bool
```

Comment:
I suppose if your AWS credentials don't have permissions to do this, that would be one use case. But in that case, S3_SECRET_ACCESS_KEY wouldn't exist either, since that's supposed to be the credentials with access to the S3 bucket. Offhand this doesn't seem like a common case... but I suppose it doesn't hurt anything. That should be ok.